src/components/BaseSeparatePage/index.module.scss

```diff
@@ -76,11 +76,12 @@
   }
 
   .supportsWrap {
-    padding-bottom: 80px;
     border-bottom: 0.5px solid;
   }
 
   @media (max-width: $mobileBreakPoint) {
+    overflow: hidden;
```

Comment:
Currently there is already `overflow: hidden` above, this should be left when the rebase, remove it.

Reply:
ok